mhkit/wave/performance.py

```diff
@@ -1,7 +1,10 @@
 import numpy as np
 import pandas as pd
+import xarray as xr
 import types
 from scipy.stats import binned_statistic_2d as _binned_statistic_2d
+from mhkit import wave
```

Comment:
Needed to import wave module as script references resource and graphics module outside of performance module.